samples/powerpoint/preview-apis/insert-slides.yaml

```diff
@@ -0,0 +1,137 @@
+                const selectedSlideID = await getSelectedSlideID();
+
+                context.presentation.insertSlidesFromBase64(chosenFileBase64, {
+                    formatting: PowerPoint.InsertSlideFormatting.useDestinationTheme,
+                    targetSlideId: selectedSlideID + "#"
+                });
+                await context.sync();
+            });
+        }
+
+        function getSelectedSlideID() {
+            // Wrap a call of one of the Common APIs in a Promise-returning
+            // function, so that it can be easily called within a run() method
+            // of an application-specific API.
+            return new OfficeExtension.Promise<string>(function(resolve, reject) {
+                Office.context.document.getSelectedDataAsync(Office.CoercionType.SlideRange, function(asyncResult) {
+                    try {
+                        if (asyncResult.status === Office.AsyncResultStatus.Failed) {
+                            reject(console.error(asyncResult.error.message));
+                        } else {
+                            resolve(asyncResult.value.slides[0].id);
+                        }
+                    } 
+                    catch (error) {
+                        reject(console.log(error));
+                    }
+                });
+            });
+        }
+
+        /** Default helper for invoking an action and handling errors. */
+        async function tryCatch(callback) {
+            try {
+                await callback();
+            } 
+            catch (error) {
+                // Note: In a production add-in, you'd want to notify the user through your add-in's UI.
+                console.error(error);
+            }
+        }
+    language: typescript
+template:
+    content: |-
+        <section class="ms-font-m">
+            <p>This sample shows how to insert slides from another presentation into the currrent presentation.</p>
+        </section>
+
+        <section class="samples ms-font-m">
+            <h3>Try it out</h3>
+            <p>
+                <ol>
+                    <li>Open this add-in in a brand new presentation.</li>
+                    <li>Add at least 2 slides to the presentation.</li>
+                    <li>Next, select a PowerPoint presentation from which to insert slides.</li>
+                </ol>
+                <form>
+                    <input type="file" id="file" />
+          </form>
+            </p>
+            <p>Press <b>Insert all slides</b> to insert all the slides from the source presentation at the beginning of the
+                current presentation using the source formatting.</p>
+            <button id="insert-all-slides" class="ms-Button">
+                    <span class="ms-Button-label">Insert all slides</span>
+            </button>
+            <p>Press <b>Insert after selected slide</b> to insert the slides from the source presentation just after the selected
```

Comment:
Do we need these long descriptions? I think button names like "Insert after selected slide" is sufficiently descriptive.